### ISSUE TITLE
Cannot build for Varnish 4.1 using branch 4.1

### ISSUE THREAD

**Issue by pinkeen** [gquintard/libvmod-accept#3]
The `4.1` branch requires varnish 5.0+ which causes `./configure` to fail.

```
checking for Varnish... 4.1.11
configure: error: Varnish version 5.0 or higher is required.
```

Upon changing the `VARNISH_PREREQ([5.0])` constraint in `configure.ac` it has been built successfully. Is it only the matter of a wrong constraint?

**Comment by pinkeen** [comment]
I've changed this constraint, prepared RPMs, added a feature based on this to our provisioning (custom language-detection redirects) and it seems to be working flawlessly :)
Thanks for your work!
(Issue is still valid though, 4.1 branch doesn't build for varnish 4.1 OOTB)

**Comment by gquintard** [comment]
Hi, there's a 4.1 branch that should build without issue.

Note that 4.1 is EOL anyway, you should upgrade

**Comment by pinkeen** [comment]
Hello, thanks for the feedback, however, I think the PR/issue still makes sense as 4.1 branch requires varnish 5.0 and won't build against varnish 4.x - see https://github.com/gquintard/libvmod-accept/blob/4.1/configure.ac#L31.

I've checked varnish site (http://varnish-cache.org/releases/) and according to it varnish 4.1 is still supported. Unfortunately upgrade is not something I can do lightly due to complicated VCL to maintain that also depends on vmods which are not available for Varnish 5.0+

**Comment by gquintard** [comment]
Hi,

> I've checked varnish site (http://varnish-cache.org/releases/) and according to it varnish 4.1 is still supported. 

4.1 should be EOL: https://varnish-cache.org/lists/pipermail/varnish-announce/2018-December/000732.html Looks like the site needs updating, thanks for noticing!

> Hello, thanks for the feedback, however, I think the PR/issue still makes sense as 4.1 branch requires varnish 5.0 and won't build against varnish 4.x - see https://github.com/gquintard/libvmod-accept/blob/4.1/configure.ac#L31.

you are terribly right, I merge the PR, thanks for pushing and proving me wrong
